site/ar_LB.all.json

```diff
@@ -581,16 +581,16 @@
     "donate_button_text": {
       "other": "يتبرع"
     },
-    "shopping_error_plan_already_owned": { "other": "You already have this plan." },
-    "shopping_error_plan_expired": { "other": "This plan is no longer available." },
+    "shopping_error_plan_already_owned": { "other": "لديك بالفعل هذه الخطة" },
+    "shopping_error_plan_expired": { "other": "هذه الخطة لم تعد متوفرة" },
 
-    "wcag_aria_label_letterboxd": { "other": "View on Letterboxd" },
+    "wcag_aria_label_letterboxd": { "other": "عرض على Letterboxd" },
 
-    "header_banner": { "other": "ABC Cinemas – 21st  Film Festival, June 1 – 6, 2021" },
-    "app_badge_title": { "other": "Download the app to view your purchased content!" },
-    "app_badge_ios": { "other": "Download on the App Store" },
-    "app_badge_android": { "other": "Get it on Google Play" },
+    "header_banner": { "other": "سينما ABC - مهرجان الفيلم الحادي والعشرون ، 1-6 يونيو 2021" },
```

Comment:
not really a language specific issue but seems wrong to chuck in ABC / 2021 in the default banner text. prolly should just be a generic "welcome to our festival" message? I suppose end game would be the clients will likely have their own messaging for whatever context they wanna use it for and then turn it on in admin or something so maybe its not a big deal

Reply:
Yeah... this core translation is never going to be used... if a client has it turned on, they need their own text in there anyway.